.rubocop.yml

```diff
@@ -16,3 +16,7 @@ inherit_mode:
 #
 # See https://github.com/alphagov/rubocop-govuk/blob/main/CONTRIBUTING.md
 # **************************************************************
+
+Lint/MissingSuper:
```

Comment:
Would it be possible to update the commit message headline for this commit, just to make it clearer what is being changed.

Reply:
Or I wonder if the override can be specified in the class itself, to avoid adding an override here?

Reply:
I've dropped the commit and moved the override inline. I have a feeling that there's no way to win with this.